<ui version="4.0" >
 <class>MessengerOptionsClass</class>
 <widget class="QWidget" name="MessengerOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>208</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbFonts" >
     <property name="title" >
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblChatLabel" >
        <property name="text" >
         <string>Chat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="lblChatFont" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="pbtChatFont" >
        <property name="text" >
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="lblMessageLabel" >
        <property name="text" >
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="lblMessageFont" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QPushButton" name="pbtMessageFont" >
        <property name="text" >
         <string>Change</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbUseTabWindow" >
     <property name="text" >
      <string>Use tab window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbViewShowHtml" >
     <property name="text" >
      <string>Show HTML</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbViewShowDateTime" >
     <property name="text" >
      <string>Show date and time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbChatShowStatus" >
     <property name="text" >
      <string>Show status in chat window</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="lblSendKey" >
       <property name="text" >
        <string>Send message key sequence:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lneSendKey" />
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
